pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -105,8 +105,18 @@ func makeSimpleImportSpans(
 		}
 
 		spanCoverStart := len(cover)
-
 		for layer := range backups {
+			isCovered, err := data.coveredByLaterReintroduction(span.Key, backups[layer].EndTime)
+			if err != nil {
+				return nil, err
+			}
+			if isCovered {
+				// Don't use this backup to cover this span if the span was reintroduced
```

Comment:
I like this comment, but I'll like it even more with an example of the invalid data you're mentioning. Maybe copy paste it from the issue you've opened?